lib/fibonacci.rb

```diff
@@ -5,8 +5,25 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the the number of values in the array
+# Space complexity: O(c), where c is the integers that are being rewritten
```

Comment:
The space complexity is actually O(1) since it does not increase with the size of n.